src/controller/order/order.js

```diff
@@ -0,0 +1,91 @@
+import express from 'express';
+import pool from '../../config/config.js'
+import {ordervalidation } from '../../validation/order.validation';
+
+
+
+const createNewOrder=(req,res)=>{
+        const {error}=ordervalidation(req.body);
+        if(error)return res.status(400).json({
+            status:400,
+            error:error.details[0].message
+            })
+
+        
+
+        const owner=req.user_token.user_token;
+        const owner_id=owner.id;
+        const car_data=[
+            owner_id,
+            req.body.car_id,
+            req.body.amount
+        ];
+
+        const check_query = 'SELECT * FROM users WHERE id=$1';
```

Comment:
@emmanueltct you have two variables named the exact same way. `check-query` on line 24 and 29. This is  bound to cause errors in your code. Give them names based on what they contain.